Taxi/Taxi/Presenter/CalendarView.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  CalendarView.swift
+//  Taxi
+//
+//  Created by sanghyo on 2022/06/09.
+//
+
+import SwiftUI
+
+struct CalendarView: View {
+    @State private var currentDate = Date()
+    @State private var selectedDate = Date()
+    @State private var currentMonth = 0
+    private let today = Date()
+    private let taxiParties: [TaxiParty] = TaxiPartyMockData.mockData
+    private let action: () -> Void
+    let days = ["일", "월", "화", "수", "목", "금", "토"]
+    let intDateConverter = IntDateConverter()
+
+    init(action: @escaping () -> Void) {
+        self.action = action
+    }
+
+    var body: some View {
+        VStack {
+            VStack(spacing: 20) {
+                navigation
+                dayOfWeek
+            }
+            datePicker
+        }
+        .frame(maxWidth: .infinity)
+    }
+
+    // MARK: - view property
+
+    var navigation: some View {
+        HStack {
+            Text("\(currentDate.formattedString)")
+            Spacer()
+            Button {
+                withAnimation {
+                    currentMonth -= 1
+                }
+            } label: {
+                Image(systemName: "chevron.left")
+                    .tint(.black)
+            }
+            Button {
+                withAnimation {
+                    currentMonth += 1
+                }
+            } label: {
+                Image(systemName: "chevron.right")
+                    .tint(.black)
+            }
+        }
+        .padding(.horizontal, 10)
+        .onChange(of: currentMonth) {_ in
+            currentDate = changeCurrentMonth()
+        }
+    }
+
+    var dayOfWeek: some View {
+        return HStack {
```

Comment:
위에 navigation에서 return을 안 썼으니까 여기서도 안 쓰는게 좋을 것 같습니다!
```suggestion
        HStack {
```